packages/syft/src/syft/service/action/action_object.py

```diff
@@ -536,7 +539,7 @@ def request(self, client):
         return client.api.services.request.submit(submit_request)
 
     def _syft_try_to_save_to_store(self, obj) -> None:
-        if self.syft_node_uid is None:
+        if self.syft_node_uid is None or self.syft_client_verify_key is None:
```

Comment:
Q: Do we not save the object to store , if either uid or verify key is None ?

Reply:
Yeah, we don't save to store this case, cause it means someone has created this object locally.